src/components/Contacts/ContactDetails/ContactReferralTab/ContactReferralTab.test.tsx

```diff
@@ -1,6 +1,12 @@
+/* eslint-disable react/react-in-jsx-scope */
```

Comment:
Let's just disable this rule. It used to be needed but importing `React` is no longer needed. https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html

You can add `'react/react-in-jsx-scope': 'off',` to the `rules` section of https://github.com/CruGlobal/mpdx-react/blob/main/.eslintrc.js#L18-L33 instead of disabling this one file.

Reply:
Or continuing to import it works too I guess. We can disable it in another PR.